microceph/cmd/microceph/cluster_list.go

```diff
@@ -22,6 +24,8 @@ func (c *cmdClusterList) Command() *cobra.Command {
 		RunE:  c.Run,
 	}
 
+	cmd.Flags().StringVarP(&c.flagFormat, "format", "f", "table", i18n.G("Format (csv|json|table|yaml|compact)")+"``")
```

Comment:
Nit: suggest to use lxdCmd.TableFormatTable instead of the literal "table", just in case.